pkg/emulator/libretro/image/scale.go

```diff
@@ -1,7 +1,9 @@
 package image
 
 import (
+	"golang.org/x/image/draw"
 	"image"
+	"image/color"
 )
 
 const (
```

Comment:
Golang has something like 
https://github.com/golang/go/wiki/Iota. I'm ok with the current, just share with you some Go info :P.